### PR TITLE
Move log merger to this repo

### PR DESCRIPTION
**Pull request by maxlou05**

### CODE REVIEW

**Comment by maxlou05** [comment]
This PR depends upon the changes from #1 to be implemented

File: modules/merger/config_logger.yaml
Comment (Xierumeng) [inline modules/merger/config_logger.yaml:1]: Isn't this already in repository common?

**Review by Xierumeng — COMMENTED** [review]
~~Please add the submodule~~ I am bad at reading.

**Review by Xierumeng — APPROVED** [review]
Approved, thanks!

**Pull request merged**